contracts/Core/BlockManager.sol

```diff
@@ -67,49 +67,21 @@ contract BlockManager is Initializable, ACL, BlockStorage {
         parameters = IParameters(parametersAddress);
     }
 
-    function getBlock(uint256 epoch) external view returns(Structs.Block memory _block) {
+    function getBlock(uint256 epoch) external view returns(Structs.Block memory) {
         return(blocks[epoch]);
     }
 
-    function getBlockMedians(uint256 epoch) external view returns(uint256[] memory _blockMedians) {
```

Comment:
These getters were removed as they are no longer being used in giveRewards()
and as their purpose can be easily fulfilled by getBlock

Reply:
Please create another issue and link it to this PR.

Reply:
sure

Reply:
done https://github.com/razor-network/contracts/issues/225